app/views/layouts/districts/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Districts Homepage</h1>
+<body>
```

Comment:
Why is there an h1 tag outside the body tag? Why is an index.html.erb in the layouts folder?